src/Conda.jl

```diff
@@ -369,4 +371,46 @@ function import_list(io::IO, args...; kwargs...)
     end
 end
 
+"""
+    pip_interop(bool::Bool, env::Environment=$ROOTENV)
+
+Sets the `pip_interop_enabled` value to bool.
+If `true` then the conda solver is allowed to interact with non-conda-installed python packages.
+"""
+function pip_interop(bool::Bool, env::Environment=ROOTENV)
+    runconda(`config --set pip_interop_enabled $bool --file $(conda_rc(env))`, env)
+end
+
+"""
+    pip_interop(env::Environment=$ROOTENV)
+
+Gets the `pip_interop_enabled` value from the conda config.
+"""
+function pip_interop(env::Environment=ROOTENV)
+    dict = parseconda(`config --get pip_interop_enabled --file $(conda_rc(env))`, env)["get"]
```

Comment:
what if it doesn't have a `"get"` entry?

Reply:
same thing that would happen here
https://github.com/JuliaPy/Conda.jl/blob/5bff153b1748f7735c33206c6f34720b5994cff6/src/Conda.jl#L297-L300

Reply:
@stevengj I think that calling `parseconda config --get ...` should ensure that it will have a `get` entry